cluster/gce/trusty/helper.sh

```diff
@@ -30,7 +30,9 @@ source "${KUBE_ROOT}/cluster/gce/debian/helper.sh"
 # $1: template name (required)
 function create-node-instance-template {
   local template_name="$1"
+  sed "s/##KUBELET_PORT##/${KUBELET_PORT}/g" ${KUBE_ROOT}/cluster/gce/trusty/node_template.yaml > ${KUBE_ROOT}/cluster/gce/trusty/node.yaml
```

Comment:
See how "TEST_CLUSTER" was added and used in https://github.com/kubernetes/kubernetes/pull/13008 as an example.